frontend/src/utils/signin-token/index.js

```diff
@@ -0,0 +1,16 @@
+import * as workerTimers from 'worker-timers'
+
+const checkForTokenInMilliseconds = 2700000
+
+const SignInToken = {
+  refresh: (googleApi, success) => {
```

Comment:
Not a blocker but you as you mentioned, you _could_ have this return a promise and avoid the success callback

Reply:
Yeah, I think I will make that change.